Magento2/Sniffs/Legacy/ModuleXMLSniff.php

```diff
@@ -54,51 +53,30 @@ public function process(File $phpcsFile, $stackPtr)
             );
         }
 
-        $foundElements = $xml->xpath('/config/module');
-        if ($foundElements === false) {
-            return;
-        }
-        
-        foreach ($foundElements as $element) {
-            if (!$this->elementIsCurrentlySniffedLine($element, $stackPtr)) {
-                continue;
-            }
-            
-            if (property_exists($element->attributes(), 'version')) {
+        $foundElements = $xml->xpath('/config/module[@version]');
+        if ($foundElements !== false) {
```

Comment:
Complexity can be reduced by:
```
        if ($foundElements === false) {
            return;
        }
```

Reply:
@sivaschenko this would ignore wrong `active` attributes that are checked in the next block

Reply:
You are right, resolved.